hibike/devices/YogiBear/motor.cpp

```diff
@@ -67,13 +67,14 @@ void drive(float target) {
 void clearFault() {
   digitalWrite(INA, !digitalRead(INA));
   digitalWrite(INB, !digitalRead(INB));
-  
+
   digitalWrite(INA, !digitalRead(INA));
   digitalWrite(INB, !digitalRead(INB));
 
   Timer1.pwm(PWM, 0);
+  pinMode(enable_pin, OUTPUT);
   digitalWrite(enable_pin, LOW);
```

Comment:
you may want to throw in a few millisecond's worth of delay here.  an experimental test will tell you.